### ISSUE TITLE
Airspeed Velocity TODOs

### ISSUE THREAD

**Issue by FabioLuporini** [devitocodes/devito#1245]
In chronological order:

- [ ] Land inception PR #1074 
- [ ] Set up the GH Workflow to run on a dedicated node
- [ ] Pinning (ASV seems to provide an option for `asv run ...` for pinning. It's worth a shot?)
- [ ] Add more benchmarks (TTI, elastic, viscoelastic, .....)
- [ ] Add benchmark for compiler performance (how long does `tti_example.py -so 12` take to generate the Operator?)
- [ ] Performance regression on GPU
  - Set up the GH Workflow to run on a node with a **given** GPU attached
  - Add benchmarks for running on GPUs
- [ ] Performance regression with MPI
  - ...

**Comment by mloubout** [comment]
Isn't this all done with thematrix now?

**Comment by FabioLuporini** [comment]
Indeed. But we still need to double check TheMatrix is working properly, no fluctuations, etc. Anyway, no objections to closing this one, up to you I'd say

**Comment by rhodrin** [comment]
- [ ] `asv-results` requires purging and recreated in the proper manner, that is only the relevant asv related files and no devito files.

**Comment by FabioLuporini** [comment]
I think it's safe to close this one @rhodrin @mloubout
